src/features/mapWidget/ui/MapWidget.tsx

```diff
@@ -0,0 +1,22 @@
+import { APIProvider, Map } from "@vis.gl/react-google-maps";
+import { ReactElement } from "react";
+import { useGetMarkersQuery } from "../api/markersApi";
+import { MapMarker } from "@/entities/mapMarker/mapMarker";
+
+const mapId = import.meta.env.VITE_APP_GOOGLE_MAP_ID as string;
+const apiKey = import.meta.env.VITE_APP_GOOGLE_MAP_API_KEY as string;
+
+export function MapWidget(): ReactElement {
+  const position = { lat: 53.9, lng: 27.56667 };
+  const { data: markers = {features: []}, isLoading } = useGetMarkersQuery();
```

Comment:
It is important to remember about error handling. You can at least log it for now (in this component or in query)